<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hw1</class>
 <widget class="QMainWindow" name="hw1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Image</string>
    </property>
    <widget class="QPushButton" name="pushButton0_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load floder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton0_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton0_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_R</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Calibration</string>
    </property>
    <widget class="QPushButton" name="pushButton1_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 find corners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton1_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 find Intrinsic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton1_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.5 Show Result</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton1_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Find Distortion</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox1_1">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>99</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>1.3 Find extrinsic</string>
     </property>
     <widget class="QPushButton" name="pushButton1_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1.3 Find extrinsic</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>201</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Augmented Reality</string>
    </property>
    <widget class="QPushButton" name="pushButton2_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Show Words on Board</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton2_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Show Words Vertically</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Stereo Disparity Map</string>
    </property>
    <widget class="QPushButton" name="pushButton3_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Stereo Disparity Map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>191</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>5. VGG19</string>
    </property>
    <widget class="QPushButton" name="pushButton5_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton5_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.1 Show Agumented Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton5_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.2 Show Model Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton5_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.3 Show Accuracy and Loss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton5_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.4 Inference</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Predict =</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_img">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>128</width>
       <height>128</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>4. SIFT</string>
    </property>
    <widget class="QPushButton" name="pushButton4_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton4_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton4_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Keypoints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton4_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Matched Keypoints</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
